static/openapi/vwrs-spec.json

```diff
@@ -1390,10 +910,10 @@
             "name": "pattern",
             "in": "query",
             "required": false,
-            "description": "The search pattern of the key_id.",
+            "description": "Fetch api key information that matches the pattern. For example, a pattern could be 'aa' or 'test'. To get all api keys, set the pattern to '*'.",
```

Comment:
@diana-macrometa do yiou think we should add the "for example.." part?
Always found it strange that this was here.  How does the example help alleviate any confusion?  I even don't like the '*' since we already have it defined in the "default": field.